projects/plugins/crm/phpcs.security.xml.dist

```diff
@@ -14,7 +14,6 @@
 	<exclude-pattern>./vendor/*</exclude-pattern>
```

Comment:
FYI, I think this file is no longer needed. PHPCS is done at the monorepo level now.

Reply:
Yeah, that makes sense. I guess we just need to remember to copy

```
<rule ref="WordPress.Security.EscapeOutput">
    <properties>
        <property name="customAutoEscapedFunctions" type="array" value="zbsLink,jpcrm_esc_link"/>
    </properties>
</rule>
```

... to `projects/plugins/crm/.phpcs.dir.xml` but that's about it. I'll do it in a separate PR - thank you!